README.md

```diff
@@ -30,54 +30,48 @@
-(happy and horror) of taking equity comp, eg., why should we learn more, why should we
-care?)
+We *can not* guarantee, by any means, that you will be as fortunate as one of the first 3,000 employees of Facebook. On the other hand, we firmly believe you will be able to walk away from reading this Guide in its entirety or even one relevant section with knowledge that will enable you to avoid making very costly mistakes.
 
-🚧 Groundwork:
-Some history of the rise of equity compensation in the US and how it links to focus on
-shareholder value.
-Any links about positive/negative/controversial aspects of this broad economic trend?
+The landscape of negotiating for equity—also referred to as stock or ownership—is ridden with confusing terminology and high-stakes decisions, but it is has also been successfully navigated by many who, too, felt lost when faced with a short window in which to learn how to evaluate or deliver a job offer with an equity component. Having navigated the territory of equity compensation from the perspective of employees, founders, and lawyers—some times successfully and others unsuccessfully—we have created this Guide to help others get a fair shake, understand their decisions, and save time doing so.
+
+We'll cover all the complexities later, but the general idea is that an employee is either given a portion of a business or, much more commonly, given the right to purchase a portion of a business with the assumption that the business will grow over time, and the value of the employee's portion of the business will appreciate like an extremely rare fine French wine.
+
+Offering equity as compensation was originally conceived of to enable cash-poor startups to compete for talent against much larger and mature cash-flush businesses. In order to lure employees away from cushy salaries and bonuses, startups needed a sufficiently lucrative reward to offer their early employees.
+
+For a very small number of people, having stock options in an early stage startup can lead to enormous wealth creation. For most others, stock options end up worthless. There is a Silicon Valley narrative that you join a startup, work there for a few years, the company turns into Uber (a huge success), you cash out your options, and you're set for life. Founders of new startups frequently sell this narrative to new hires, but most startups die.
+
+👆 Despite a [well-perpetuated myth that 90–99% of startups fail](https://www.youtube.com/watch?v=Dr_pIfX0_tA), according to [a report](http://fortune.com/2017/06/27/startup-advice-data-failure/) by Cambridge Associates and Fortune Magazine, between 1990 and 2010, the percentage of companies backed by venture capital firms that failed (meaning they returned less than 1x the original investment to their investors) did not exceed 60%. Even of the remaining 40%, few go on to make their employees as rich as the truly exceptional companies like Apple, Microsoft, Amazon, Google, and Facebook.
```

Comment:
I think this paragraph from VC belabors the startup point a bit. We don't want the intro to encourage readers to self-select via company type this early, right? This Guide is not exclusively about equity comp at startups, and it's not exclusively or even primarily about how to make as much money from your equity as possible. I think the general entry scenario of "I just got offered equity I'm so rich riiiite??" is covered in preceding paragraph--We could even just move this down to Evaluating Equity Comp section if we want to keep it (it is good material!).